frontend/package.json

```diff
@@ -38,6 +39,7 @@
     "redux-logger": "3.0.6",
     "redux-thunk": "2.3.0",
     "styled-components": "5.1.0",
+    "supercluster": "7.1.0",
```

Comment:
This is the main dependency added to cluster points on the map. The other additions are supporting dependencies and typings for TypeScript